aptos-move/aptos-vm/src/sharded_block_executor/cross_shard_client.rs

```diff
@@ -113,8 +113,8 @@ impl CrossShardCommitSender {
         txn_output: &AptosTransactionOutput,
     ) {
         let edges = self.dependent_edges.get(&txn_idx).unwrap();
-        // TODO(skedia): This doesn't work for sequantial execution - fix it.
-        let write_set = txn_output.committed_output().unwrap().write_set();
+        let output = txn_output.get_cloned_output();
```

Comment:
how about avoiding the clones as described [here](https://github.com/aptos-labs/aptos-core/pull/8747#discussion_r1247029085)?